symphony/lib/boot/func.utilities.php

```diff
@@ -8,32 +8,32 @@
  * Redirects the browser to a specified location. Safer than using a
  * direct header() call
  *
- *  @param string $url
+ *  @param string $location
+ *  @param string $status (optional)
  */
-function redirect ($url)
+function redirect($location, $status = '302 Found')
 {
-    // Just make sure.
-    $url = str_replace('Location:', null, $url);
-
     if (headers_sent($filename, $line)) {
-        echo "<h1>Error: Cannot redirect to <a href=\"$url\">$url</a></h1><p>Output has already started in $filename on line $line</p>";
-        exit;
-    }
 
-    // convert idn back to ascii for redirect
+        // throw exception if headers already sent
```

Comment:
What do you mean exactly? I try to stick with `PSR-2` wherever possible, otherwise aiming for better code readability.